src/inflation/adapter/load_files.clj

```diff
@@ -0,0 +1,38 @@
+(ns inflation.adapter.load_files
+  (:require
+   [clojure.string :as str]
+   [datomic.api :as d]
+   [inflation.db.datomic.db :as db]
+   [inflation.model.price :as model]
+   [clojure.data.csv :as csv]
+   [clojure.java.io :as io]))
+
+(def conn (db/connection))
```

Comment:
Avoid using vars to handle state. Prefer passing it as parameter to the functions and create the instance at application start up. If you start to have more stuff, consider using something like component (https://github.com/stuartsierra/component).

Reply:
Yes, I made a 1:1 today and my friend commented to me about this library.
I need to understand how to always get the new datomic instance, so I don't have problems with updates in the db